R/dice.R

```diff
@@ -163,21 +172,31 @@ dice <- function(x, fm = NULL, SPC = TRUE, pctMissing = FALSE, fill = FALSE, str
   
   # safe wrapper to base::seq
   # NA in from/to (hz depths) not allowed
-  # TODO: from == to (0-thickness) horizons ??
   .seq.safe <- function(from, to, by) {
-    if(any(is.na(from)) | any(is.na(to))) {
+    if (any(is.na(from)) | any(is.na(to))) {
       stop('corrupt horizonation, consider using `strict = TRUE`', call. = FALSE)
     }
-    
+    if (from > to) {
```

Comment:
Can you please comment on what this is doing?

Reply:
I think it was sufficient to drop the 0 thickness horizons. The inversion issue had to do with sequences from 1 to 0 I believe